src/main/java/space/obminyashka/items_exchange/service/RoleService.java

```diff
@@ -18,4 +18,8 @@ public class RoleService {
     public Optional<Role> getRole(String roleName) {
         return roleRepository.findByName(roleName);
     }
+
+    public void updateUserRole(String username) {
```

Comment:
The name of the method doesn't correspond to its behavior. For the current name, it should be the second parameter **role** in it.